### DISCUSSION TITLE
Beginner Authentication Question

### DISCUSSION THREAD

**Discussion by Jez-2Sheds** [tweepy/tweepy#1582] [Questions] [answered]
Just started with this. Im confused by the apparent contradiction in the documentation.

In 'Getting started' 

https://docs.tweepy.org/en/v3.10.0/getting_started.html

a v simple 2 line OAuth1 solution is offered but then in the more detailed Authentication page on OAuth1 a far more detailed approach is described. Can someone confirm that all of the info on the OAuth1 guide 

https://docs.tweepy.org/en/v3.10.0/auth_tutorial.html 

relates only to a web application that requires a callback URL

**Comment by Harmon758** [comment, marked as answer]
Yes, the OAuth 1a Authentication section of the Authentication Tutorial is specifically for a user-facing application.
If you have your access token and access token secret, the example in the Hello Tweepy section on the Getting started page is all you need to do.

For clarification/simplification of this documentation, there's an open issue; see #769.
